### ISSUE TITLE
brokers property of KafkaConfig can receive a function

### ISSUE THREAD

**Issue by smhmayboudi** [nestjs/nest#5633]
## Is your feature request related to a problem? Please describe.

There is a sample code at https://kafka.js.org/docs/configuration which broker can accept function.

## Describe the solution you'd like

```ts
export type BrokersFunction = () => string[] | Promise<string[]>

export interface KafkaConfig {
  brokers: string[] | BrokersFunction
```

## What is the motivation / use case for changing the behavior?

It needs to connect rest-proxy to receive the broker list then connect to the brokers.

**Comment by kamilmysliwiec** [comment]
Would you like to create a PR for this issue?

**Comment by kamilmysliwiec** [comment]
Fixed in 7.5.0
